File: src/main/kotlin/application/controller/dialogs/rule/ruleloader/RuleLoaderDialog.java
package application.controller.dialogs.rule.ruleloader;

import application.controller.dialogs.rule.RuleWidget;
import application.model.rules.ruleloader.RuleLoader;

public class RuleLoaderDialog extends RuleWidget {
    public RuleLoaderDialog() {
        super();

        ruleFamily = new RuleLoader();
    }

    @Override
    public void updateRule(String rulestring) {
        ruleFamily.setRulestring(rulestring);
    }
}

File: src/main/kotlin/application/controller/dialogs/rule/ruleloader/RuleLoaderDialog.kt
package application.controller.dialogs.rule.ruleloader

import application.controller.dialogs.rule.RuleWidget
import application.model.rules.ruleloader.RuleLoader

class RuleLoaderDialog : RuleWidget() {
    override fun updateRule(rulestring: String?) {
        ruleFamily!!.rulestring = rulestring
    }

    init {
        ruleFamily = RuleLoader()
    }
}